### ISSUE TITLE
say what syck does in the README

### ISSUE THREAD

**Issue by tpo** [ruby/syck#32]
tlrd; please say what syck does in the README

So I have a Gemfile here, with 'syck' in it and when I run the project syck complains about something. So first thing I do is to search the net for what use syck actually has: https://duckduckgo.com/?q=syck+gem

But all that shows me is "A gemified version of Syck from Ruby's stdlib. Syck has been removed from Ruby's stdlib, and this gem is meant to bridge the gap for people that haven't updated their YAML yet."

Heureka! Syck does something that something else once did, which doesn't exist anymore and because it doesn't exist anymore it's actually not trivial to find info about what that thing that once existed but doesn't anymore actually did and so I know just as little as before what syck actually is good for...

Oh well, so in my never ending search for wisdom I scroll down and find the gem:

```
SYNOPSYS

require 'syck'
Syck.dump %w{ foo bar baz }
```

Oh a riddle? We now know that the code above probably can be parsed and executed! Yay, I can see clearly now! But uuuh, what's `Syck.dump %w{ foo bar baz }` actually *supposed to do*... ?

OK, I cut my rant frenzy taking control of me. Now seriously: wouldn't it be really useful to so say in the README of a project what the project's actually good for? Guess: "syck adds a generic fallback method that will try to produce a readable representation of any variable"?

**Comment by rubyFeedback** [comment]
Agreed on the docu problem. This is really a huge problem for ruby in general.

In regards to the syntax: psych works in many ways similar to syck, so Psych.dump()
is yielding the same output (at the least for that given input).

I used to use syck because I had non-unicode files. I eventually managed to transition
into unicode. But I always appreciated better documentation. This is really a big problem
in the ruby world. Many japanese devs don't like writing english documentation so it
is a chicken-egg problem (there are exceptions; people such as mame have near 
fluent native english language skills and provide a lot of documentation in their own
gems).
